setup.py

```diff
@@ -10,7 +10,7 @@
 VERSION = '0.1.0'
 DESCRIPTION = 'Kaolin: A PyTorch library for accelerating 3D deep learning research'
 URL = 'https://github.com/NVIDIAGameWorks/kaolin'
-AUTHOR = ''
+AUTHOR = 'Nvidia'
```

Comment:
Should double-check whether we write `Nvidia` or `NVIDIA`

Reply:
Based on here: https://nvidia.frontify.com/d/qZxUpjTCnt96/nvidia-logo-quickstart-guide, I believe you are correct. Good catch!